conan/tools/gnu/pkgconfigdeps/pc_files_creator.py

```diff
@@ -0,0 +1,76 @@
+from conan.tools.gnu.pkgconfigdeps.pc_files_templates import PCFilesTemplate
+from conan.tools.gnu.pkgconfigdeps.pc_info_loader import PCInfoLoader
+
+
+class PCFilesCreator(object):
+
+    def __init__(self, conanfile, dep):
+        self._conanfile = conanfile
```

Comment:
This conanfile seems to be the "consumer" conanfile. And it is only used here for the ``description`` field. Shouldn't the description field be from the actual ``dep``? Maybe it is even possible to remove it completely, and have the ``description`` almost empty or automated to something like "Conan package: xxxx"